### ISSUE TITLE
Eval op with unknown ns yields two "done" responses

### ISSUE THREAD

**Issue by eerohele** [nrepl/nrepl#215]
## Steps to reproduce the problem

1. Send `{"id": 1, "op": "eval", "code": "(inc 1)", "ns": "unknown.ns", "session": "1d54c31a-f4bb-4700-9618-1296b343e3b9"}`.

The nREPL server returns:

```
{"id": 1, "ns": "unknown.ns", "session": "status": ["namespace-not-found", "done", "error"], "session": "1d54c31a-f4bb-4700-9618-1296b343e3b9"}
{"id": 1, "status": ["done"], "session": "1d54c31a-f4bb-4700-9618-1296b343e3b9"}
```

[Looking](https://github.com/nrepl/nrepl/blob/255d8420ec0c14397f29b906e1ce7cefd544a274/src/clojure/nrepl/middleware/interruptible_eval.clj#L75-L76) at the [code](https://github.com/nrepl/nrepl/blob/255d8420ec0c14397f29b906e1ce7cefd544a274/src/clojure/nrepl/middleware/interruptible_eval.clj#L150-L153), it seems obvious how this happens. How to fix it is less obvious.

I think as far as clients are concerned, it'd be easiest if we could avoid sending the second `done` message.

## Environment & Version information

nREPL 0.8.2

### Clojure version

Clojure 1.10.1

### Java version

OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)

### Operating system

macOS

**Comment by bbatsov** [comment]
> I think as far as clients are concerned, it'd be easiest if we could avoid sending the second done message.

Yeah, I totally agree. 

I'm assuming the bug you discovered is a result of the introduction of streamed value responses, as originally the `done` on successful `eval` was part of the only `value` response you'd get back then.

**Comment by bbatsov** [comment]
Looking at the code it seems that just removing the `:done` from the error message would be enough, but I don't like that this will break the common pattern of buckling the done status with whatever error status there is (if any).

**Comment by eerohele** [comment]
>I'm assuming the bug you discovered is a result of the introduction of streamed value responses, as originally the `done` on successful `eval` was part of the only `value` response you'd get back then.

Yeah. The issue this bug causes in my client is purely cosmetic: currently, in my client, `done` causes a newline to get printed. This bug just yields two newlines instead of one.

>Looking at the code it seems that just removing the `:done` from the error message would be enough, but I don't like that this will break the common pattern of buckling the done status with whatever error status there is (if any).

I had a quick look at the code earlier, and I agree: that would be the simplest way to fix the issue, but might not be the best. Not sure how much work the alternative solution would involve.
